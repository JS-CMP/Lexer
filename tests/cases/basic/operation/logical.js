var a = true && false
var b = true || false
var c = !a
console.log(a, b, c)
