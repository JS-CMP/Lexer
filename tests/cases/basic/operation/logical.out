#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"

int main() {JS::Any a=JS::Any(true)&&JS::Any(false);JS::Any b=JS::Any(true)||JS::Any(false);JS::Any c=!a;console["log"](a,b,c);}
